pkg/reconciler/knativeserving/knativeserving.go

```diff
@@ -159,13 +160,25 @@ func (r *Reconciler) reconcile(ctx context.Context, ks *servingv1alpha1.KnativeS
 
 // Transform the resources
 func (r *Reconciler) transform(ctx context.Context, instance *servingv1alpha1.KnativeServing) (mf.Manifest, error) {
-	logger := logging.FromContext(ctx)
+	log := logging.FromContext(ctx)
```

Comment:
nit: I'd prefer if those stayed "logger", they're like this in the upstream at least.

Reply:
done